### PR TITLE
Add link to QubesOS guide

### PR DESCRIPTION
**Pull request by cd2357**
Update Downloads page with link to Qubes setup guide.

### CODE REVIEW

File: _data/en.yml [hunk 1]
```diff
@@ -153,6 +153,8 @@ downloads:
   tdRedHat: Red Hat/Fedora
   tdArchLinux: Arch Linux
   tdArchLinuxText: Arch User Repository
+  tdQubes: QubesOS
```
Comment (m52go) [inline _data/en.yml]:
```suggestion
  tdQubes: Qubes OS
```

File: _data/en.yml [hunk 2]
```diff
@@ -153,6 +153,8 @@ downloads:
...
+  tdQubesText: QubesOS Setup Guide
```
Comment (m52go) [inline _data/en.yml]:
```suggestion
  tdQubesText: Qubes OS Setup Guide
```

**Review by m52go — COMMENTED** [review]
Looks good to me, thanks for adding.

Nitpick - it seems like Qubes officially brands their name as "Qubes OS" (with a space).

Shall we use that instead?

**Comment by cd2357** [comment]
Good point, done.

**Comment by m52go** [comment]
ACK

**Pull request merged**